runtime/engine/pom.xml

```diff
@@ -27,7 +27,7 @@
     <maven.compiler.source>11</maven.compiler.source>
     <maven.compiler.target>11</maven.compiler.target>
     <jacoco.coverage.ratio>0.70</jacoco.coverage.ratio>
-    <jacoco.missed.count>4</jacoco.missed.count>
+    <jacoco.missed.count>7</jacoco.missed.count>
```

Comment:
Does this missed count reduce back to 4 again when we have all the unit tests in place?